### PR TITLE
Avoid methods defined in rake task exposing globally

### PR DESCRIPTION
**Pull request by serco-chen**
I was using thinking-sphinx with this gem but somehow I cannot run `rake ts:rebuild` to rebuild the core index.
And later I find out it was because of the method `config` defined in `gettext:po_to_json` exposed.

It would be glad if someone could review this PR or offer a better way to deal with this issue.

### CODE REVIEW

**Comment by tboerger** [comment]
Thank you for the pull request. The method files_to_translate MUST be defined like before, it have to overwrite the method of gettext_i18n_rails. Can you move the full functionality into a class named GettextI18nRailsJs::Task? Than ich can coverage it with tests and we are fully out of global scope. Please watch for the Rubocop rules, they are currently failing on TravisCI.

**Comment by serco-chen** [comment]
@tboerger 
Done. Thanks for the reply.

File: lib/gettext_i18n_rails_js/task.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+require "gettext_i18n_rails/tasks"
+
+module GettextI18nRailsJs
+  module Task
+    extend self
+
+    def files_list
+      Pathname.glob(
+        ::File.join(
+          locale_path,
```
Comment (tboerger) [inline lib/gettext_i18n_rails_js/task.rb]: I think this is maybe not defined here, is that possible?

Comment (serco-chen) [inline lib/gettext_i18n_rails_js/task.rb]: @tboerger No, I have required `gettext_i18n_rails/tasks`, so it's not a problem.

**Comment by tboerger** [comment]
Sorry, i totally forgot this pull request. I will merge it now and on top of that i will add some more sugar before i release the new version.

**Comment by serco-chen** [comment]
Awesome, looking forward to the new version.

**Comment by tboerger** [comment]
Already released it yesterday :)

**Pull request merged**